src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -355,30 +355,54 @@
        vec
        (sorted-map :name environment :modules)))
 
+(def ModuleList
```

Comment:
Yep apologizes....I had some extra debugging functions that I accidentally committed. They should be removed now!